xml/schema/Price/PriceFieldValue.xml

```diff
@@ -6,6 +6,13 @@
   <name>civicrm_price_field_value</name>
   <add>3.3</add>
   <component>CiviContribute</component>
+  <paths>
+    <add>civicrm/admin/price/field/option/edit?reset=1&amp;action=add&amp;fid=[PriceFieldValue_PriceField_price_field_id_01.id]&amp;sid=[PriceFieldValue_PriceField_price_field_id_01.price_set_id]</add>
+    <view>civicrm/admin/price/field/option/edit?reset=1&amp;action=view&amp;oid=[id]&amp;fid=[price_field_id]&amp;sid=[PriceFieldValue_PriceField_price_field_id_01.price_set_id]</view>
+    <update>civicrm/admin/price/field/option/edit?reset=1&amp;action=update&amp;oid=[id]&amp;fid=[price_field_id]&amp;sid=[PriceFieldValue_PriceField_price_field_id_01.price_set_id]</update>
+    <delete>civicrm/admin/price/field/option/edit?reset=1&amp;action=delete&amp;oid=[id]&amp;fid=[price_field_id]&amp;sid=[PriceFieldValue_PriceField_price_field_id_01.price_set_id]</delete>
+    <browse>civicrm/admin/price/field/option</browse>
```

Comment:
@aydun it looks like this path is referencing an explicit join. Explicit joins don't have a stable name that can be referenced like this (since the name of the join is anything you want it to be when you call `->addJoin()` in the API. Is it possible to do this with an implicit join, like `[price_field_id.price_set_id]`?

Reply:
That does work for the view, update and delete links but not for the `add` one.  Any chance you could tweak how things are processed for the add button to make that work?

Reply:
The ADD button processes tokens differently than links, because it doesn't have any row data to use (it's not in a row) the only data it has for tokens is the incoming filters. I know this works for a few other pages, e.g Custom Fields has an incoming filter by `custom_group_id` and that is passed through to the ADD button.
So if you switch your page filter to use implicit-join-style syntax then they should get passed through that way to the button.

Reply:
Hmm - I can't get it to work for that one.  Maybe there's something else not quite right in the metadata?  I've changed it as you suggest.  How about we merge this then come back to the add button when another PR with the new pages?